moduleframework/common.py

```diff
@@ -889,14 +828,14 @@ def get_module_type_base():
     if module_type not in get_backend_list():
         parent = get_config().get("module", {}).get(module_type, {}).get("parent")
         if not parent:
-            raise ModuleFrameworkException("Module (%s) does not provide parent backend parameter (there are: %s)" %
+            raise mtfexceptions.ModuleFrameworkException("Module (%s) does not provide parent backend parameter (there are: %s)" %
                                            (module_type, get_backend_list()))
     if parent not in get_backend_list():
-        raise ModuleFrameworkException("As parent is allowed just base type: %s" % get_backend_list)
+        raise mtfexceptions.ModuleFrameworkException("As parent is allowed just base type: %s" % get_backend_list)
     return parent
 
 
-def get_docker_file(dir_name=DEFAULT_DIR_OF_DOCKER_RELATED_STUFF):
+def get_docker_file(dir_name=conf["docker"]["dockerfiledefaultlocation"]):
```

Comment:
This can be None if `dockerfiledefaultlocation` is empty. Do you count with it?

Reply:
it will lead to traceback, it is expected